wait/sql.go

```diff
@@ -88,6 +89,11 @@ func (w *waitForSql) WaitUntilReady(ctx context.Context, target StrategyTarget)
 		return fmt.Errorf("sql.Open: %v", err)
 	}
 	defer db.Close()
+
+ 	db.SetConnMaxLifetime(0)
+	db.SetMaxIdleConns(3)
+	db.SetMaxOpenConns(3)
```

Comment:
Although the code LGTM, I'm a bit reluctant to hardcode this value here without giving the user the ability to set them in the wait strategy, according to their own requirements for the database under test. If we add them as part of the SQL strategy, then I think it's fine.

Reply:
Thinking about this twice, these settings should be added in a separate PR. Will send a commit on top removing them, and opening a discussion to make this configurable